keyboards/sofle/choc/config.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2022 Brian Low
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+// USB Device descriptor parameter
+#define VENDOR_ID       0xFC32 // Copied from sofle/rev1 so that Via works
```

Comment:
if the matrix is different, then youd need a different json for via

Reply:
same key matrix, different LED configuration